gb2260/division.py

```diff
@@ -119,10 +119,8 @@ def stack(self):
 def make_year_key(year):
     """A key generator for sorting years."""
     if year is None:
-        return (LATEST_YEAR, 12)
+        return (LATEST_YEAR, 7)
     year = str(year)
-    if len(year) == 4:
-        return (int(year), 12)
```

Comment:
See [spec](https://github.com/cn/GB2260/blob/develop/spec.md)
In the Data Structure part,"revision: (str) The revision year (month).",
Atfer all,i think it is more safe to reserve the code to deal year.